### ISSUE TITLE
change namelist for climate forcing

### ISSUE THREAD

**Issue by JessicaNeedham** [NGEET/tutorial-containers#9]
We need to make the following changes to the namelist definition to get the GSWP forcing data to work for out single site:
```
diff --git a/components/data_comps/datm/cime_config/namelist_definition_datm.xml b/components/data_comps/datm/cime_config/namelist_definition_datm.xml
index 66689d1c93..aeb56fb5e1 100644
--- a/components/data_comps/datm/cime_config/namelist_definition_datm.xml
+++ b/components/data_comps/datm/cime_config/namelist_definition_datm.xml
@@ -1573,9 +1573,8 @@
         FLDS     lwdn
       </value>
       <value  stream="CLM1PT.ELM_USRDAT">
-        ZBOT     z
         TBOT     tbot
-        RH       rh
+       QBOT     shum
         WIND     wind
         PRECTmms precn
         FSDS     swdn
lines 1-15/15 (END)
```

**Comment by glemieux** [comment]
This is addressed with 41501e18cf618a389a88eb8f3ac8d0bbf9aa647a and 2da79f94ebbd6515e3f3c3ab8ed2c40fc7030e8b.  

@JessicaNeedham if we have attendees who want to use their own forcing data, will the fact that we've modified this be an issue?

**Comment by glemieux** [comment]
I realized that we could actually handle this via the fates-tutorial inventory creation script.  I've added that there via https://github.com/NGEET/fates-tutorial/commit/6d5dcb719680758703206446cb723993c0315725.
